tests/views/test_collection_instrument_view.py

```diff
@@ -154,6 +156,11 @@ def test_upload_collection_instrument_if_survey_does_not_exist(self):
     @requests_mock.mock()
     def test_collection_instrument_upload_with_ru(self, mock_bucket, mock_request):
         mock_request.post(url_collection_instrument_link_url, status_code=200)
+        mock_request.get(
+            "http://localhost:8080/surveys/db0711c3-0ac8-41d3-ae0e-567e5ea1ef87",
+            status_code=200,
+            json={"surveyId": "db0711c3-0ac8-41d3-ae0e-567e5ea1ef87", "surveyRef": "139"},
+        )
```

Comment:
This chunk is repeated 3 times, we should try to extract this. This is also what I meant about writing inline as well